cmd/program/program.go

```diff
@@ -13,6 +13,7 @@ import (
 	tea "github.com/charmbracelet/bubbletea"
```

Comment:
General comment:
There are some conflicts in program.go with main. Otherwise, I tested the code, and everything is working. When you have time, resolve the conflict, and we are good to go.